lib/widgets/no_events_overlay.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class NoEventsOverlay extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(color: Colors.white),
+      child: Center(
+        child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
+          Icon(Icons.calendar_today, size: 100, color: Colors.grey),
+          Text(
+            'No events',
+            textDirection: TextDirection.ltr,
+            style: TextStyle(
+              fontSize: 32,
+              color: Colors.grey,
```

Comment:
🤔 Could we pick an equivalent color from the theme? So it would play nicely with dark theme then.

Reply:
Seems like we can detect dark mode and use different color accordingly. https://stackoverflow.com/questions/57475630/detect-if-ios13-dark-mode-is-enabled-from-flutter-dart/58156366#58156366 
What do you think?